specification/nginx/Nginx.Management/tspconfig.yaml

```diff
@@ -0,0 +1,49 @@
+output-dir: "{project-root}/"
+parameters:
+  "service-dir":
+    default: "sdk/nginx"
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    use-read-only-status-schema: true
+    emitter-output-dir: "{project-root}/.."
+    azure-resource-provider-folder: "resource-manager"
+    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/openapi.json"
+  "@azure-tools/typespec-csharp":
+    flavor: azure
+    package-dir: "Azure.ResourceManager.Nginx.NginxPlus"
+    clear-output-folder: true
+    model-namespace: false
+    namespace: "{package-dir}"
+  "@azure-tools/typespec-python":
+    flavor: azure
+    package-dir: "azure-mgmt-nginxplus"
+    namespace: "azure.mgmt.nginxplus"
+    generate-test: true
+    generate-sample: true
+  "@azure-tools/typespec-java":
+    flavor: azure
+    package-dir: "azure-resourcemanager-nginxplus"
+    namespace: "com.azure.resourcemanager.nginxplus"
+    service-name: "nginx"
+  "@azure-tools/typespec-ts":
+    experimental-extensible-enums: true
+    package-dir: "arm-nginxplus"
+    flavor: azure
+    package-details:
+      name: "@azure/arm-nginxplus"
+  "@azure-tools/typespec-go":
+    service-dir: "sdk/resourcemanager/nginx"
+    package-dir: "armnginx"
+    module: "github.com/Azure/azure-sdk-for-go/{service-dir}/{package-dir}"
+    fix-const-stuttering: true
+    generate-samples: true
+    generate-fakes: true
+    head-as-boolean: true
+    inject-spans: true
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-rulesets/resource-manager"
+  disable:
+    "@azure-tools/typespec-azure-core/casing-style": "Allow uppercase namespaces for our RP"
```

Comment:
why?

Reply:
Our namespace name has always been all uppercase but typespec complains about it, emphasizing that it should be Pascal case. Trying to change it to Pascal case opened a can of worms where we now had to rewrite all our swaggers in Typespec and had tons of breaking changes reported. So with the advice of the Typespec team, I turned off the casing style check. See [this conversation](https://teams.microsoft.com/l/message/19:906c1efbbec54dc8949ac736633e6bdf@thread.skype/1750870555949?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1750775090620&teamName=Azure%20SDK&channelName=TypeSpec%20Discussion&createdTime=1750870555949) for reference